src/Findologic/Request/FindologicRequestFactory.php

```diff
@@ -6,6 +6,7 @@
 
 use FINDOLOGIC\Api\Definitions\OutputAdapter;
 use FINDOLOGIC\Api\Exceptions\InvalidParamException;
+use FINDOLOGIC\Api\Requests\Request as FindoLogicApiRequest;
```

Comment:
It is not `FindoLogic`, but `Findologic`
```suggestion
use FINDOLOGIC\Api\Requests\Request as FindologicApiRequest;
```